doc/build/build.md

```diff
@@ -85,9 +85,8 @@ cmake .. \
          -DDNNL_AARCH64_USE_ACL=ON \
          <extra build options>
 ~~~
-Only ACL versions 20.11 or above are supported. Using ACL versions above	
-20.11 may require the `-DCMAKE_CXX_STANDARD=14` and `-DCMAKE_CXX_EXTENSIONS=OFF`
-flags to be passed.
+
+Only ACL versions 21.02 or above are supported.
```

Comment:
Please move information on the minimal supported ACL version to README.md. Not a blocker for this particular MR though.

Reply:
Thanks for the feedback! Done